__tests__/api.test.js

```diff
@@ -49,4 +49,61 @@ describe("GET: API endpoints", () =>{
         })
     })
 
-})
\ No newline at end of file
+})
+
+
+describe("GET /api/articles/:article_id", () =>{
+    test('GET:an array of objects with properties author, title, article_id, body', ()=>{
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then((response) =>{
+                const article= response.body.article
+                console.log(article)
+                expect(typeof article.author).toBe('string')
+                expect(typeof article.title).toBe('string')
+                expect(typeof article.article_id).toBe('number')
+                expect(typeof article.body).toBe('string')
+                expect(typeof article.topic).toBe('string')
+                expect(typeof article.created_at).toBe('string')
+                expect(typeof article.votes).toBe('number')
+                expect(typeof article.article_img_url).toBe('string')
+            
+        })
+    })
+
+    test("GET:404 if article_id is not valid number", ()=>{
+        return request(app)
+        .get('/api/articles/invalidNumber')
+        .expect(400)
+        .then((response) =>{
+            expect(response.body.msg).toBe('Invalid point')
+            })
+        })
+
+}) 
+
+describe('GET /api/articles', ()=>{
+    test('GET:AN OBJECT common_count, total count of all comments in each article_id ', ()=>{
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then ((response) =>{
+            expect(Array.isArray(response.body.articles)).toBe(true)
+            response.body.articles.forEach((article) =>{
+                expect(typeof article.comment_count).toBe('string')
+            })
+        })
```

Comment:
And you may want to check all the properties of the objects to make sure it following a certain shape